cpp/CMakeLists.txt

```diff
@@ -118,7 +118,12 @@ endfunction(bbp_enable_clang_formatting)
 
 function(bbp_enable_cmake_formatting)
   find_package(PythonInterp 3.5 REQUIRED)
-  find_package(CMakeFormat 0.6.0 EXACT REQUIRED)
+  find_package(CMakeFormat 0.6 REQUIRED)
+  if(CMakeFormat_FOUND AND CMakeFormat_VERSION VERSION_GREATER_EQUAL 0.6)
```

Comment:
Should this not be `VERSION_GREATER` to get `>` rather than `≥`?

Reply:
Also, this seems to be a feature of CMake 3.7… `cmake_minimum_required` only ever requires 3.6 in this repo

Reply:
> Should this not be `VERSION_GREATER` to get `>` rather than `≥`?

Thanks for spotting this, I used 0.6 for testing purpose. I meant 0.7

Reply:
> Also, this seems to be a feature of CMake 3.7… `cmake_minimum_required` only ever requires 3.6 in this repo

Very good point indeed. I will bump cmake_minimum_required to 3.10 in a next PR anyway (the version shipped with ubuntu bionic), so we are good here. But thanks for checking this.

Reply:
OK, works for me!